.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Could you add these file from vscode to gitignore, and remove them.

Just add `.vscode/*` to the end of the gitignore file.